### ISSUE TITLE
[BC break] --reporter option overrides first the consoleReporter in monitor:health

### ISSUE THREAD

**Issue by dav-m85** [liip/LiipMonitorBundle#100]
```monitor:health --reporter=...``` currently add a new reporter to the existing pre-configured ```console``` reporter. This one cannot be avoided. It would be better to have a preconfigured reporter, and override this reporter by using the ```--reporter``` option.

In short, the first use of ```--reporter``` would replace the console reporter, and subsequent use would behave the same, adding reporters to the stack.

**Comment by lsmith77** [comment]
I guess we cannot break BC here .. but we could provide a new parameter ``--overwrite-reporter``

/cc @kbond

**Comment by kbond** [comment]
Sorry, this one slipped through the cracks.

@dav-m85, to clarify: when running `app/console monitor:health --reporter=my_reporter`, you only want `my_reporter` to run? What is the issue with the console reporter running as well?

**Comment by dav-m85** [comment]
It would be cleaner in my opinion. A possible use case would be:
```
# if console runs, this one produce wrong xml
console monitor:health --reporter=xml > myhealthreport.xml
```
tl;dr; if I want full access to stdout when running monitor:health, i can't.

My first idea about this issue was to move the --nagios option to a reporter, but well it is working quite well like this, adding --quiet option to silence completly (not PR-ed yet).

**Comment by kbond** [comment]
Ok fair enough.

Would `console monitor:health --quiet --reporter=xml > myhealthreport.xml` not work then?

**Comment by dav-m85** [comment]
Yes, if the reporter uses echo or file://stdout instead of the OutputInterface instance providen in the command. But I could live with that.

If that's ok for you, then let's close the issue :) Thanks !

**Comment by kbond** [comment]
There is no way to provide the reporter with the OutputInterface instance from the command.

**Comment by dav-m85** [comment]
Gosh. You're right.
